worker/src/main/java/gov/cms/ab2d/worker/processor/domainmodel/PatientClaimsRequest.java

```diff
@@ -0,0 +1,43 @@
+package gov.cms.ab2d.worker.processor.domainmodel;
+
+import com.newrelic.api.agent.Token;
+import gov.cms.ab2d.worker.adapter.bluebutton.GetPatientsByContractResponse;
+import gov.cms.ab2d.worker.processor.StreamHelper;
+
+import java.time.OffsetDateTime;
+
+public class PatientClaimsRequest {
```

Comment:
Can we use Lombok to auto-generate the constructor and the getters?

Reply:
done